currency_rate_update/views/res_currency_rate.xml

```diff
@@ -10,10 +10,7 @@
         <field name="inherit_id" ref="base.view_currency_rate_form" />
         <field name="arch" type="xml">
             <form position="inside">
-                <div class="oe_chatter">
-                    <field name="message_follower_ids" widget="mail_followers" />
-                    <field name="message_ids" widget="mail_thread" />
-                </div>
+                <chatter />
```

Comment:
```suggestion
                <chatter/>
```

Reply:
Formatting is related with pre-commit

Reply:
Whoah, what a load of bollocks, as if I didn't have enough reason to hate pre-commit :expressionless: 
My bad, didn't know that's how it formats in those cases :]